src/pybind/mgr/cephadm/services/nvmeof.py

```diff
@@ -36,14 +36,17 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
 
         # TODO: check if we can force jinja2 to generate dicts with double quotes instead of using json.dumps
         transport_tcp_options = json.dumps(spec.transport_tcp_options) if spec.transport_tcp_options else None
+        name = '{}.{}'.format(utils.name_to_config_section('nvmeof'), igw_id)
```

Comment:
Probably unrelated to this change, but it might be good to fix this up sooner rather than later, but the var name `igw_id` makes no sense to me.

Reply:
yeah, I think that's leftover from the nvmeof implementation being mostly copied from the iscsi one. I can add a commit to rename the variable as well.